src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -699,9 +699,7 @@ private ValueTask<VariantDefinition> AssignVariantAsync(EvaluationEvent evaluati
 
         private void BindSettings(IFeatureFilterMetadata filter, FeatureFilterEvaluationContext context, int filterIndex)
         {
-            IFilterParametersBinder binder = filter as IFilterParametersBinder;
-
-            if (binder == null)
+            if (!(filter is IFilterParametersBinder binder))
```

Comment:
Is this change caused by any linting rule?

Reply:
Yeah this was shown to me by code analysis. I can't find the rule- even if I undo the change, it's no longer flagging it... 

The check right below this uses this format anyway, so we should be consistent anyway.

Reply:
> so we should be consistent anyway

Do you mean to revert this change or to keep it?

Reply:
Either keep this change- or change the following if statement to use the same style as this one.